ipatests/test_integration/test_installation.py

```diff
@@ -1255,8 +1254,12 @@ def test_kra_certs_renewal(self):
         """
         Test that the KRA subsystem certificates renew properly
         """
-        kra = krainstance.KRAInstance(self.master.domain.realm)
-        for nickname in kra.tracking_reqs:
+        tracking_reqs = {
+            'auditSigningCert cert-pki-kra': 'caAuditSigningCert',
+            'transportCert cert-pki-kra': 'caTransportCert',
+            'storageCert cert-pki-kra': 'caStorageCert',
+        }
```

Comment:
I also don't like code duplication. How about you move the information to a common place like ``ipalib.constants`` and import the definition from there?

Reply:
I understand your concern, therefore I've moved aforementioned code to the `constants`.

Reply:
Marking resolved as it is resolved.